VAMobile/src/api/payments/getPayments.tsx

```diff
@@ -35,8 +37,11 @@ const getPayments = async (year: string | undefined, page: number): Promise<Paym
  * Returns a query for user payments history
  */
 export const usePayments = (year: string | undefined, page: number, options?: { enabled?: boolean }) => {
+  const { data: authorizedServices } = useAuthorizedServices()
+  const queryEnabled = options && has(options, 'enabled') ? options.enabled : true
   return useQuery({
     ...options,
+    enabled: !!(authorizedServices?.paymentHistory && queryEnabled),
```

Comment:
This query should also be dependent on whether payments is in downtime

```tsx
const paymentsInDowntime = useDowntime(DowntimeFeatureTypeConstants.payments)
...
enabled: !!(authorizedServices?.paymentHistory && !paymentsInDowntime && queryEnabled),
```

Reply:
Oh yeah i was going to ask whether downtime should be moved as well. Will make the change :+1: